.github/workflows/release.yml

```diff
@@ -0,0 +1,62 @@
+name: Create version packages PR or release
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch: # Allow manual triggering of the workflow
+    inputs:
+      tag:
+        description: "Tag to release"
+        required: true
```

Comment:
You can actually drop this if you want, as you can select the tag when running the workflow.